src/mlpack/methods/cf/normalization/combined_normalization.hpp

```diff
@@ -0,0 +1,215 @@
+/**
+ * @file combined_normalization.hpp
+ * @author Wenhao Huang
+ *
+ * CombinedNormalization is a class template for performing a sequence of data
+ * normalization methods which are specified by template parameter.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_CF_NORMALIZATION_COMBINED_NORMALIZATION_HPP
+#define MLPACK_METHODS_CF_NORMALIZATION_COMBINED_NORMALIZATION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace cf {
+
+/**
+ * This normalization class performs a sequence of normalization methods on
+ * raw ratings.
```

Comment:
Can you add a simple example in the comments here something like:

https://github.com/mlpack/mlpack/blob/6a59dd568f0b4c62af52e9813bbaaecc15d8a95d/src/mlpack/methods/quic_svd/quic_svd.hpp#L39-L51